### PR TITLE
Add apt/history logs

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR adds the following files to the driver which is helpful for support and engineering/QA:

- /var/log/apt/history.log
- /var/log/apt/history.log.1.gz
- /var/log/apt/term.log
- /var/log/apt/term.log.1.gz

Tested on a `galp3-b` and a `lemp9`. One was running Ubuntu 20.04 LTS and the other running Pop 20.10.

### CODE REVIEW

**Review by bflanagin — COMMENTED** [review]
Checked on `lemp9` (20.04, 20.10) and `gaze15` (20.10) works as expected. Not sure how many more systems need to be tested before approval. Will ask and approve if ~4 is enough.

**Comment by leviport** [comment]
> Not sure how many more systems need to be tested before approval.
That should be sufficient. I'm happy with it if you are.

**Review by bflanagin — APPROVED** [review]

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**
